Sources/ArgumentParser/Parsing/Interactive.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  File.swift
+//
+//
+//  Created by Keith  on 2022/6/21.
+//
```

Comment:
Let's get the correct header comment on this file.

Reply:
done